src/ytdl_sub/entries/variables/soundcloud_variables.py

```diff
@@ -25,6 +25,17 @@ def track_number_padded(self) -> str:
         """
         return f"{self.track_number:02d}"
 
+    @property
+    def total_tracks(self) -> int:
+        """
+        Returns
+        -------
+        The total tracks in album. For singles, it will always be 1.
+        """
+        if self.kwargs_contains("total_tracks"):
```

Comment:
the kwargs here is referring to the YTDL dict returned for the soundcloud entry. I don't think total_tracks is there. We can always return 1 here just like we do with `track_number`

Reply:
Yes, didn't release what kwargs was actually doing, changed it, to that implementation.